### ISSUE TITLE
Dap Window Customisation

### ISSUE THREAD

**Issue by axieax** [mfussenegger/nvim-jdtls#157]
### Problem Statement

Hi, I would like to customise my workflow so that running a JUnit test will bring up both the test results summary as well as the stdout from the program in separate windows. Currently, I've configured dap.defaults.fallback.terminal_win_cmd = "30vsplit new" so that the stdout appears on the right side in a vsplit window. Would it be possible to position the summary (repl) on the side as well, preferably above the stdout?

**Comment by axieax** [comment]
Oh I see thanks. Do you know if there a way to configure nvim-dap to set this as a default?

**Comment by mfussenegger** [comment]
No, but `repl.open` is intended to be called via keybinding, and if you define one you can just hardcode the value you want there.

There is no place within nvim-dap where the repl is opened automatically. (nvim-dap-ui does that, and it would be up to that extension to provide a default value)
